Description: Change warning message (SDL joystick not available) to VERBOSE
Author: Richard Goedeken <Richard@fascinationsoftware.com>
Origin: upstream, https://bitbucket.org/richard42/mupen64plus-input-sdl/changeset/de6c9330ffe3

---
diff --git a/src/config.c b/src/config.c
index 885c30cf28508361190027972ac8ac6283f57948..b4d1ead0869d23ff2a6a32316c0168ac9810c5f3 100644
--- a/src/config.c
+++ b/src/config.c
@@ -505,7 +505,7 @@ void load_configuration(int bPrintSummary)
             {
                 controller[n64CtrlIdx].device = DEVICE_AUTO;
                 controller[n64CtrlIdx].control->Present = 0;
-                DebugMessage(M64MSG_WARNING, "N64 Controller #%i: Disabled, SDL joystick %i is not available", n64CtrlIdx+1, sdlCtrlIdx-1);
+                DebugMessage(M64MSG_VERBOSE, "N64 Controller #%i: Disabled, SDL joystick %i is not available", n64CtrlIdx+1, sdlCtrlIdx-1);
             }
             else
             {
